adafruit_pyportal/network.py

```diff
@@ -89,22 +90,21 @@ def __init__(
         self._image_resize = image_resize
         self._image_position = image_position
         self._image_dim_json_path = image_dim_json_path
-
+        self._secrets = secrets_data
```

Comment:
This shouldn't be necessary because it is already being set in the base class.

Reply:
Thanks! I ran into a CI issue, but I think that was because your PortalBase PR hadn't been merged yet.